test/test-app/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can you add a simple README.md to the "test-app" folder explaining what this app is used for? Honestly just pointing to https://github.com/Azure/azure-functions-durable-js/issues/353 might be enough

Reply:
Done